test/fixtures/transpile-css-prop-add-import/output.js

```diff
@@ -27,7 +27,7 @@ function _templateObject() {
 
 function _taggedTemplateLiteral(strings, raw) { if (!raw) { raw = strings.slice(0); } return Object.freeze(Object.defineProperties(strings, { raw: { value: Object.freeze(raw) } })); }
 
-var _StyledDiv = _styledComponents["default"].div(_templateObject());
+var _StyledDiv = (0, _styledComponents["default"])("div")(_templateObject());
```

Comment:
I'm not sure why Babel is adding this sequence (`(0, fn)`), seems unnecessary.

Reply:
Apparently it's a strict-mode thing: https://stackoverflow.com/questions/32275135/why-does-babel-rewrite-imported-function-call-to-0-fn